.github/workflows/browsers-local.yml

```diff
@@ -0,0 +1,125 @@
+name: Browsers - local
```

Comment:
```suggestion
name: Browser Tests
```

It may not be Browserstack, but I wouldn't call it local either.

Reply:
Changed